internal/command/command.go

```diff
@@ -126,6 +128,11 @@ func (c Command) AddToParent(parent *cobra.Command) {
 			checkVersion(logger)
 		}
 
+		// check contract migrations if flag is set
+		if !Flags.SkipContractMigrationCheck {
+			checkContractMigrations(state, logger, flow)
```

Comment:
I think you may encounter a nil pointer dereference here.

Notice how the error may be swallowed when initializing the flowkit state https://github.com/onflow/flow-cli/pull/1547/files#diff-69d3575d337e6861d23e877e04bbdb99f437a82661f706f055a50e82b1e8282aR111-R113.  If someone runs this comand & doesn't have a flow.json present, I'd hazard to guess that this ends up in a panic state.

Reply:
pushed changes so that we check to see if `state` is not nilil first. If it is nil, we just return, so no panic